things/api/things/http/responses.go

```diff
@@ -220,13 +220,29 @@ func (res connectionRes) Code() int {
 }
 
 func (res connectionRes) Headers() map[string]string {
-	return map[string]string{}
+	return map[string]string{
+		"Warning-Deprecated": "This endpoint will be depreciated in 0.11.0. It will be replaced with the bulk endpoint found at /connect.",
```

Comment:
We will release v1.0.0  just after 0.10.0 you can see the plan [here](https://github.com/mainflux/mainflux/milestones)

Reply:
I'll switch these messages to 1.0.0.  My thoughts were that the new bulk endpoints would be released in 0.10.0 and the single endpoints removed in the following release.  This would give any users time to switch over.